RiotSharp.AspNetCore/RiotSharp.AspNetCore.csproj

```diff
@@ -1,7 +1,23 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netstandard1.3</TargetFramework>
+    <PackageId>RiotSharp.AspNetCore</PackageId>
+    <Version>1.0.0</Version>
+    <PackageVersion>1.0.0</PackageVersion>
```

Comment:
It's the first major release, so I think 1.0.0 is matching. See [NuGet docs](https://docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages#semantic-versioning) for reference.